README.md

```diff
@@ -5,6 +5,13 @@
 A [webpack](https://webpack.js.org) loader for [svelte](https://svelte.technology).
 
 
+## Install
+
+```
+npm install --save svelte svelte-loader
```

Comment:
```suggestion
npm install svelte-loader
```
Svelte-loader requires svelte as a peer dependency, if you haven't already, run `npm install svelte`